### DISCUSSION TITLE
Cant wait for blender 4.0 update

### DISCUSSION THREAD

**Discussion by Jenssons** [bnpr/Malt#507] [General]
thats all on the title i just cant wait for the update for the newest version of blender big fan of this engine, keep up the amazing work

**Comment by pragma37** [comment]
Thanks for the nice words.
The release is ready for download. :)

The update was a bit more complex than usual this time.

**Reply by Jenssons** [reply]
HYPE

**Comment by Jenssons** [comment]
Now I wait for 4.0 update of my favourite blender addon

**Reply by pragma37** [reply]
OOOPS! I forgot to update the Release page message (I'll do it ASAP), but the latest releases ARE for 4.0.
f7e8a9258f827f3d181ba369b73f8bc3e5d74e8f

**Reply by Jenssons** [reply]
awesome!
